cnf-certification-test/networking/suite.go

```diff
@@ -259,3 +268,52 @@ func testDualStackServices(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("Found %d services in error, check error log for details", len(errorServices)))
 	}
 }
+
+const (
+	NFTables = "nftables"
+	IPTables = "iptables"
+)
+
+func TestIsConfigPresent(env *provider.TestEnvironment, name string) {
+	var badContainers []*provider.Container
+	var errorContainers []*provider.Container
+	var function func(cut *provider.Container) (bool, error)
+	if name == NFTables {
+		function = netutil.IsNFTablesPresent
+	} else {
```

Comment:
What if the "name" passed as argument is not "nftables" or "iptables"? Maybe a switch with a default case printing an error and failing the test case could fit better here. Or just another "else if" branch for "iptables" handling the error with the generic "else".